packages/components/src/range-control/index.tsx

```diff
@@ -69,6 +69,7 @@ function UnforwardedRangeControl(
 		railColor,
 		renderTooltipContent = ( v ) => v,
 		resetFallbackValue,
+		size = 'default',
```

Comment:
For the RangeControl interface, we'll want to go straight to the `__next40pxDefaultSize` API as outlined in #46741. (Better not introduce a new `size` prop that will be obsolete in a few months!)

```suggestion
		__next40pxDefaultSize = false,
```

Reply:
`__next40pxDefaultSize` isn't available yet, is it?

Reply:
Yes, this will be the first occurrence of a `__next40pxDefaultSize` prop in the codebase. Inside of the RangeControl, NumberControl will still need to receive a `size` prop. So basically something like:

```jsx
					<InputNumber
						size={ __next40pxDefaultSize ? '__unstable-large' : 'default' }
```

Reply:
This should be in place now, but let me know how it looks!